internal/pkg/exec/errors.go

```diff
@@ -24,3 +24,19 @@ type ErrNoExpose struct {
 func (e ErrNoExpose) Error() string {
 	return fmt.Sprintf("no EXPOSE statements in Dockerfile %s", e.Dockerfile)
 }
+
+// ErrDockerCommandNotFound means the docker command is not found.
+type ErrDockerCommandNotFound struct{}
+
+func (e ErrDockerCommandNotFound) Error() string {
+	return "docker: command not found"
+}
```

Comment:
nit: can be 
```suggestion
// ErrDockerCommandNotFound means the docker command is not found.
var ErrDockerCommandNotFound  = errors.New("docker: command not found")
```

Reply:
But the IDE would complain that `ErrDockerCommandNotFound` is not a type.

Reply:
I think you might have to use `errors.Is(err, exec.ErrDockerCommandNotFound)` in this scenario

Reply:
Hmm maybe leave as is for consistency. Still good to know that!

Reply:
nvm i'll make the change.